Makefile

```diff
@@ -4,7 +4,8 @@ VERSION ?= 1.9
 export CLOVERAGE_VERSION = 1.0.11-SNAPSHOT
 
 # The test-cljs target needs to be modified if working with JDK9
-JAVA_VERSION = $(shell lein version | cut -d " " -f 5 | cut -d "." -f 1-2)
+JAVA_VERSION = $(shell lein with-profile +sysutils \
+                       sysutils :java-version-simple | cut -d " " -f 2)
```

Comment:
Btw, I would have named this "Major", not "Simple" if I were you.

Reply:
Yep, I considered that, but thought people might object that "1.7" isn't "major" in the SemVer sense. Substituting another arbitrary semantic on top of the existing one. Do you feel strongly about it? This is all new, easy to change and re-release.

Reply:
Not at all. Just random 2 cents.